src/lib/validation/formValidators/company.ts

```diff
@@ -0,0 +1,48 @@
+import logger from "./../../Logger";
+import { GenericValidator } from "./../../validation/generic";
+
+export class CompanyFormsValidator extends GenericValidator {
+
+    constructor (classParam?: string) {
+        super();
+    }
+
+    validateCreateCompany (payload: any): Promise<any> {
```

Comment:
Just wondering if the intention is to remove (after copying) any of this code that has come from the template project but is not relevant for the app that we're creating/building?

Reply:
Yep, these are irrelevant. Removed.